src/Toolkit/Pagination.php

```diff
@@ -398,11 +398,17 @@ protected function setProperties(array $props)
             $this->page = $this->firstPage();
         }
 
-        // validate page based on all params
+        // allow a page value of 1 even if there are no pages;
+        // otherwise the exception will get thrown for this pretty common case
         $min = $this->firstPage();
         $max = $this->pages();
+        if ($this->page === 1 && $max === 0) {
```

Comment:
> The section "notes" could not be loaded: Pagination page 2 does not exist, expected 0-0

This will give the error if the items in the section are reset by deleting or changing the status of the listed pages while outside the first page (Reproduce). Could something like this be more stable?

    if ($max === 0) {

or 

    if ($this->page > 0 || $max === 0) {

Reply:
If the max/total is zero, why do we check the page number? So if the max/total is zero, shouldn't the page number always be zero? 🤔

Reply:
Yes, it should be. But if we silently fix the page number without throwing an Exception, we‘ll have exactly the same situation as before where an overflow was ignored.

Reply:
I'm trying to ignore the error, and you're trying to throw it away 😅